allennlp/common/util.py

```diff
@@ -196,7 +198,13 @@ def get_spacy_model(spacy_model_name: str, pos_tags: bool, parse: bool, ner: boo
             disable.append('parser')
         if not ner:
             disable.append('ner')
-        spacy_model = spacy.load(spacy_model_name, disable=disable)
+        try:
+            spacy_model = spacy.load(spacy_model_name, disable=disable)
+        except OSError:
+            logger.warning(f"Spacy models '{spacy_model_name}' not found.  Downloading and installing.")
+            spacy_download(spacy_model_name)
```

Comment:
Maybe also catch a failed download as an error? I'd be pretty confused if I ran a model locally and within running the code, I got failed download errors. But also perhaps my comment below is a little more obvious (in terms of users).